tools/discovery/api.yaml.erb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(nelsonjr): Make all Zone and Region resource ref
+
+--- !ruby/object:Api::Product
+name: Google Cloud <%= ctx[:product].title %>
+prefix: g<%= ctx[:product].title.split[0].downcase %>
+versions:
+  - !ruby/object:Api::Product::Version
+    name: <%= ctx[:product].version %>
+    base_url: <%= ctx[:product].base_url %>
+scopes:
+    - <%= ctx[:product].scope %>
+objects:
+<% ctx[:product].resources.each do |object| -%>
+  - !ruby/object:Api::Resource
+    name: '<%= object.name.capitalize %>'
+    base_url: <%= object.base_url %>
+    virtual: <%= object.virtual %>
+    description: |
+      <%= object.schema['description'] %>
```

Comment:
Something somewhere is going to have to do line breaks for the 80 character rule :)

Reply:
Yeah, I'll have to fix this.